rules/auth0-authorization-extension.js

```diff
@@ -3,60 +3,44 @@
 *  Updated by kerin.cosford@digital.justice.gov.uk, 2017-01-14T22:47:58.651Z
  */
 function (user, context, callback) {
-  // If connection is not passwordless skip this rule
-  var connection = user.identities[0].connection;
+  const connection = user.identities[0].connection;
```

Comment:
So we can use ES6 in rules? Are we 100% sure about that?

Reply:
Yes